composer.json

```diff
@@ -6,13 +6,15 @@
     "homepage": "https://github.com/pusher/pusher-php-server",
     "license": "MIT",
     "require": {
-        "php": ">=5.2",
+        "php": ">=5.3",
```

Comment:
Maybe make this 5.3.3 since phpunit doesn't run on anything less than that. We have no way of checking pusher works on 5.3.0.